### ISSUE TITLE
Generate the random wallet

### ISSUE THREAD

**Issue by Hirama** [iron-wallet/iron#358]
Generate a new random wallet on initial start-up, as providing a pre-drained wallet can lead to user confusion.
<img width="1238" alt="image" src="https://github.com/iron-wallet/iron/assets/2853815/56ce45f4-8caa-4c24-be9b-4157cc1290ed">

**Comment by naps62** [comment]
Thanks for starting this discussion.

I think the intended direction is something different though, keeping in line with the fact this is meant for developers
What I'm imagining is the onboarding flow (which currently only asks for Alchemy API key) including the ability to opt-in to having the `test test ... junk` wallet, while also making it clear it is only for test purposes.

Switching to a randomly-generated wallet is counterproductive, as it is the wallet that is funded by default when booting anvil / hardhat.

**Comment by Hirama** [comment]
Indeed, you're correct. However, by monitoring on-chain activity, it becomes clear that this "method" is counterproductive and needs terminating, as a multitude of users are losing their test tokens. Incidentally, Hardhat does not display user wallets directly as your application does. So if it possible to add some hints that address is compromised or something else?

**Comment by naps62** [comment]
yes, adding some hints on the UI can definitely be done (and makes sense)
